toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -226,6 +226,67 @@ So, any existing ExtraCommandLine value in the base image will be replaced.
 If the partitions are not customized, then the `ExtraCommandLine` value will be appended
 to the existing `grub.cfg` file.
 
+### SELinux
+
+Specifies the mode to set SELinux to.
+
+If this field is not specified, then the existing SELinux mode in the base image is
+maintained.
+
+If SELinux is enabled, then all the file-systems that support SELinux will be have their
+file labels updated/reset (using the `setfiles` command).
+
+Supported options:
+
+- `disabled`: Disables SELinux.
+
+- `permissive`: Enables SELinux but only logs access rule violations.
+
+- `enforcing`: Enables SELinux and enforces all the access rules.
+
+- `force-enforcing`: Enables SELinux and sets it to enforcing in the kernel
+  command-line.
+  This means that SELinux can't be set to `permissive` using the `/etc/selinux/config`
+  file.
+  
+Example:
+
+```yaml
+SystemConfig:
+  KernelCommandLine:
+    SELinux: enforcing
+
+  PackagesInstall:
+  - selinux-policy
+  - selinux-policy-modules
+  - setools-console
+  - policycoreutils-python-utils
+```
+
+### SELinuxPolicy
+
+When SELinux is enabled, the Mariner Image Customizer tool ensures the image has the
+`selinux-policy` package installed.
+This package contains the default SELinux rules and is required for SELinux enabled
+images to be functional.
+
+However, some images may wish to use their own custom SELinux policy package.
+Such images can override the name of policy package using the `SELinuxPolicy` property.
+
+Example:
+
+```yaml
+SystemConfig:
+  KernelCommandLine:
+    SELinux: enforcing
+    SELinuxPolicy: my-selinux-policy
```

Comment:
why have field to specify the package name if it needs to be listed under `PackagesInstall:`? (where as if it is the default, we don't specify `SELinuxPolicy` and rely only on the user listing it under `PackagesInstall`.
Does MIC need to be aware of which policy package is in use?

Reply:
i agree, left a similar comment - this is a bit confusing and redundant
ideally, if user specifies selinux and no policy - we should install selinux-policy, or else the policy that is specified in selinuxpolicy. and for either case not have them (mandatorily) also mention it in PackagesInstall. 
(this could also be added in a future PR)

Reply:
I have changed the property to be a boolean that disables the check. This should hopefully make the config less confusing and redundant.